beheer.md

```diff
@@ -22,6 +22,40 @@ In dit document worden allerlei procedures en overzichten beschreven die van bel
 ## Lijst van deelnemende partijen en personen
 Bij de ontwikkeling van Open API standaarden nemen diverse partijen deel. Het is handig voor ontwikkelaars maar ook voor beheerders om daar een overzicht van te hebben. Daarom dient de project manager of scrum master bij de aanvang van een ontwikkelproject maar ook tijdens het project als de teamsamenstelling wijzigt de [lijst met Deelnemende partijen](https://github.com/VNG-Realisatie/api-beheer/blob/master/Deelnemende%20partijen.md) in te vullen. Ook de beheerder of beheerders van de standaard moeten na in beheername van een standaard of bij wijziging van de beheerder op deze lijst worden vermeldt.
 
+## Beheerproces
+Zodra een Open API standaard is ontwikkeld en goedgekeurd moet het worden overgedragen aan een beheerteam.
+Voor die betreffende standaard wordt dat beheerteam daarna verantwoordelijk voor:
+1. beantwoorden van vragen;
+2. het oplossen van problemen.
+
+Doorontwikkeling van de standaard hoort daar dus niet bij.
+
+*Beantwoorden van vragen*
+Voor het beantwoorden van vragen m.b.t. de betreffende standaard zijn twee kanalen beschikbaar. Ten eerste het specifiek voor de standaard beschikbaar gestelde kanaal in de Slack workspace 'VNG API Community' en ten tweede het eveneens specifiek voor de standaard beschikbaar gestelde discussie forum.
+
+_Beschouwen we de discussie voorzieningen op GitHub ook als een goed alternatief voor de discussie fora?
+Zo ja dan moeten we n.m.m. een keuze tussen beide maken. Het monitoren van meerdere omgevingen op vragen/wijzigingsverzoeken moet zo veel mogelijk voorkomen worden._
```

Comment:
Eens dat er een keuze moet komen: 2 kanalen (Github en het forum) monitoren lijkt me onwenselijk, maar de combinatie Slack en Github lijkt me prima. Slack is lekker laagdrempelig en als een discussie te hard gaat, issue voor aanmaken op Github en eventueel assignen aan de juiste persoon (domein expert, koppelvlak expert, developer, etc.).

Vervangen door Github/Gitlab issues sluit beter aan bij actief mee ontwikkelen EN je kan zien hoe je vraag is opgelost omdat (als het goed is) een fix wordt doorgevoerd en (automagisch) gekoppeld aan het betreffende issue. Dat gebeurt nu op het KING forum vrij summier of niet in proza.

Ook de link naar een intern redmine systeem voor het "echte" issue lijkt me niet passen in het proces.

Reply:
Prima, kiezen we voor GitHub/GitLab.